superset/viz.py

```diff
@@ -151,9 +151,6 @@ def get_df(self, query_obj=None):
         # If the datetime format is unix, the parse will use the corresponding
         # parsing logic.
         if df is None or df.empty:
```

Comment:
Note that the `df.empty` check is only required by the `test_get_df_returns_empty_df` unit test due to the mocking.  Otherwise it would be safe to proceed with an empty `pd.DataFrame`.